demo/src/test/java/com/workday/autoparse/json/demo/InstanceUpdaterTest.java

```diff
@@ -26,6 +30,7 @@
 import static com.workday.autoparse.json.demo.InstanceUpdaterTestUtils.CONTEXT;
 import static com.workday.autoparse.json.demo.InstanceUpdaterTestUtils.getUpdateMapFromFile;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.CoreMatchers.is;
```

Comment:
I think all the changes to the imports should go away when you use `assertEquals`. Same in the other test file.